src/main/java/com/blamejared/crafttweaker/impl/loot/modifiers/CTLootModifierManager.java

```diff
@@ -0,0 +1,174 @@
+package com.blamejared.crafttweaker.impl.loot.modifiers;
+
+import com.blamejared.crafttweaker.CraftTweaker;
+import com.blamejared.crafttweaker.api.CraftTweakerAPI;
+import com.blamejared.crafttweaker.api.annotations.ZenRegister;
+import com.blamejared.crafttweaker.api.loot.conditions.ILootCondition;
+import com.blamejared.crafttweaker.api.loot.modifiers.ILootModifier;
+import com.blamejared.crafttweaker.impl.actions.loot.ActionRegisterLootModifier;
+import com.blamejared.crafttweaker.impl.actions.loot.ActionRemoveLootModifier;
+import com.blamejared.crafttweaker.impl.loot.conditions.CTLootConditionBuilder;
+import com.blamejared.crafttweaker_annotations.annotations.Document;
+import com.google.common.collect.ImmutableMap;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.ResourceLocationException;
+import net.minecraftforge.common.ForgeInternalHandler;
+import net.minecraftforge.common.loot.IGlobalLootModifier;
+import net.minecraftforge.common.loot.LootModifierManager;
+import org.openzen.zencode.java.ZenCodeType;
+
+import java.lang.invoke.MethodHandle;
+import java.lang.invoke.MethodHandles;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@ZenRegister
+@ZenCodeType.Name("crafttweaker.api.loot.modifiers.LootModifierManager")
+@Document("vanilla/api/loot/modifiers/LootModifierManager")
+@SuppressWarnings("unused")
+public final class CTLootModifierManager {
+    public static final CTLootModifierManager LOOT_MODIFIER_MANAGER = new CTLootModifierManager();
+
+    private static final MethodHandle LMM_GETTER;
+    private static final MethodHandle LMM_MAP_GETTER;
+    private static final MethodHandle LMM_MAP_SETTER;
+
+    private CTLootModifierManager() {}
+
+    static {
+        try {
+            final MethodHandles.Lookup lookup = MethodHandles.lookup();
+
+            final Class<?> forgeInternalHandlerClass = ForgeInternalHandler.class;
+            final Method lmmGetterMethod = forgeInternalHandlerClass.getDeclaredMethod("getLootModifierManager");
+            lmmGetterMethod.setAccessible(true);
+
+            final Class<?> lmmClass = LootModifierManager.class;
+            final Field registeredLootModifiersField = lmmClass.getDeclaredField("registeredLootModifiers");
+            registeredLootModifiersField.setAccessible(true);
+
+            LMM_GETTER = lookup.unreflect(lmmGetterMethod);
+            LMM_MAP_GETTER = lookup.unreflectGetter(registeredLootModifiersField);
+            LMM_MAP_SETTER = lookup.unreflectSetter(registeredLootModifiersField);
+        } catch (ReflectiveOperationException e) {
+            throw new RuntimeException("Unable to reflect into the loot modifier manager", e);
+        }
+    }
```

Comment:
Can these be moved to `Util.make()` (the MC class)?

Prefer that over static blocks like this